samwise.js

```diff
@@ -1,6 +1,6 @@
-const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?